src/style.scss

```diff
@@ -1,3 +1,44 @@
-.App {
-    background: green;
+$base-color: #FFCD1B;
+$black-color: #202020;
+$gray-color: #606060;
+$background-color: #e5e5e5;
+$white-color: #ffffff;
+
+
+.button {
+    border: none;
+    cursor: pointer;
+     &--large {
+        background: $base-color;
+        color: $black-color;
+        padding: 6px;
+        width: 50px;
```

Comment:
What will happen if we will not want to use `iconOnly` and set a text to a button that will exceed 50px width?